.env.example

```diff
@@ -35,6 +35,20 @@ DB_LOG_SQL=false
 # Don't use a timezone offset (like +01:00) or a timezone abbreviation (like CEST)
 # TIMEZONE=Europe/Paris
 
+# Visibility of directories and (media) files in LYCHEE_UPLOADS
+# Possible values are:
+#
+#  - private: world group has no read nor write access
```

Comment:
```suggestion
#  - private: world group has no read or write access
```

Reply:
Fixed in https://github.com/LycheeOrg/Lychee/pull/1351/commits/2ed01096e40f9755488a4627a362e6204441bd6a.